central/src/components/GameDashboard.jsx

```diff
@@ -71,7 +70,7 @@ export default function GameDashboard({ loading, nextToken, games, handleScrollD
           loader={<h4>Loading...</h4>}
           height={`calc(100vh - 156px)`}
           scrollableTarget="GameDashboard"
-          style={{display: 'flex', justifyContent: 'flex-start', width: '100%', flexWrap: 'wrap', overflowY: 'scroll', zIndex: -2}}
+          className={classes.infiniteScroll}
         > 
```

Comment:
infiniteScroll is how we do the loading on scroll for the game templates (so we don't need pagination buttons or anything)